File: LeetCode/1009/solution.py
def bitwiseComplement(n: int) -> int:
        bin_n = bin(n)
        str_bin_n = str(bin_n)
        reversed_str_bin_n = ""

        for i in range(2, len(str_bin_n)):
            if str_bin_n[i] == "0":
                reversed_str_bin_n += "1"
            else:
                reversed_str_bin_n += "0"

        final_int = int(("0b" + reversed_str_bin_n), 2)

        return final_int

print(bitwiseComplement(5))

File: LeetCode/1009/solution.js
var bitwiseComplement = function(n) {
    const bin_n = n.toString(2)
    let rev_bin_n = ""

    for (let i = 0; i < bin_n.length; i++) {
        if (bin_n[i] == "1") {
            rev_bin_n += "0"
        } else {
            rev_bin_n += "1"
        }
    }

    const final_int = parseInt(rev_bin_n, 2)

    return final_int

};

console.log(bitwiseComplement(5))